src/rgw/rgw_rest_s3.cc

```diff
@@ -2882,7 +2882,7 @@ void RGWDeleteMultiObj_ObjStore_S3::send_partial_response(rgw_obj_key& key,
 
       s->formatter->dump_string("Key", key.name);
       s->formatter->dump_string("VersionId", key.instance);
-      s->formatter->dump_int("Code", r.http_ret);
+      s->formatter->dump_string("Code", r.s3_code);
```

Comment:
@mattbenjamin Thanks. 
Was just an friendly reminder if developer skipped at time of coding.